sconfig/shared/src/main/scala-2/org/ekrich/config/impl/OriginType.scala

```diff
@@ -11,6 +11,7 @@ object OriginType {
   final val FILE = new OriginType("FILE", 1)
   final val URL = new OriginType("URL", 2)
   final val RESOURCE = new OriginType("RESOURCE", 3)
+  final val ENV_VARIABLE = new OriginType("ENV_VARIABLE", 4)
```

Comment:
Note: this also has to be added to the `scala-3` real `enum`. Don't forget to add it to this line as well. https://github.com/ekrich/sconfig/blob/3379df17617cb86c7b20fd4cd3f0d59c070cf5b3/sconfig/shared/src/main/scala-2/org/ekrich/config/impl/OriginType.scala#L17